pages/common/brew-install.md

```diff
@@ -0,0 +1,16 @@
+# brew install
+
+> Install a Homebrew formula or cask.
+> More information: <https://docs.brew.sh/Manpage#install-options-formulacask->.
+
+- Install a formula/cask:
+
+`brew install {{formula|cask}}`
```

Comment:
```suggestion
`brew install {{formula_or_cask_name}}`
```

Using the `|` syntax would imply that only the strings `formula` or `cask` are accepted.

Reply:
There are a lot of pages using `|` to denote different kinds of placeholders that do not mean the only thing you can put there is the literal string used in the placeholder.
For example: `docker service inspect {{service_name|ID}}`.

In a page documenting a package manager it is quite obvious that there is more than one package to be installed.

Reply:
I agree with @acuteenvy here, we use the same (`|`) on other pages to denote multiple possible options. If we go with the new proposed option with the snake case then the placeholder will become long and would complicate the example.